examples/axp_optimizer_example.py

```diff
@@ -0,0 +1,134 @@
+import time
+import pandas as pd
+import os
+import uuid
+
+from fds.analyticsapi.engines import ApiException
+from fds.analyticsapi.engines.api.axp_optimizer_api import AXPOptimizerApi
+from fds.analyticsapi.engines.api_client import ApiClient
+from fds.analyticsapi.engines.configuration import Configuration
+from fds.analyticsapi.engines.model.axioma_equity_optimization_parameters_root import AxiomaEquityOptimizationParametersRoot
+from fds.analyticsapi.engines.model.axioma_equity_optimizer_strategy import AxiomaEquityOptimizerStrategy
+from fds.analyticsapi.engines.model.optimizer_account import OptimizerAccount
+from fds.analyticsapi.engines.model.axioma_equity_optimization_parameters import AxiomaEquityOptimizationParameters
+from fds.analyticsapi.engines.model.optimizer_output_types import OptimizerOutputTypes
+from fds.analyticsapi.engines.model.optimizer_trades_list import OptimizerTradesList
+from fds.analyticsapi.engines.model.optimization import Optimization
+
+from urllib3 import Retry
+
+host = "https://api.factset.com"
+username = os.environ["ANALYTICS_API_USERNAME_SERIAL"]
+password = os.environ["ANALYTICS_API_PASSWORD"]
+
+
+def main():
+    config = Configuration()
+    config.host = host
+    config.username = username
+    config.password = password
+    # add proxy and/or disable ssl verification according to your development environment
+    # config.proxy = "<proxyUrl>"
+    config.verify_ssl = False
+
+    # Setting configuration to retry api calls on http status codes of 429 and 503.
+    config.retries = Retry(total=3, status=3, status_forcelist=frozenset([429, 503]), backoff_factor=2,
+                           raise_on_status=False)
+
+    api_client = ApiClient(config)
+
+    try:
+        # {
+        #     "data": {
+        #         "strategy": {
+        #             "id": "Client:/Optimizer/CN_TEST"
+        #         },
+        #         "account": {
+        #             "id": "CLIENT:/OPTIMIZER/IBM.ACCT"
+        #         },
+        #         "optimization": {
+        #             "riskmodeldate": "09/01/2020",
+        #             "backtestdate": "09/01/2020",
+        #             "cashflow": "0"
+        #         },
+        #         "outputtypes": {
+        #             "trades": {
+        #                 "identifiertype": "SedolChk",
+        #                 "includecash": false
+        #             }
+        #         }
+        #     }
+        # }
+        axp_optimizer_strategy = AxiomaEquityOptimizerStrategy(id="Client:/Optimizer/CN_TEST")
```

Comment:
Can you use variables to store the values and pass them here similar to pa examples?

Reply:
hey Akanksha, in general I always prefer adding variables but these variables are not adding much value since they are only used in one location. I was irritated myself with these variables as they were just unnecessarily adding time/complexity to build examples. I only did it for consistency. But if we all are fine we can start skipping them going forward for all examples.